packages/react-interactions/accessibility/src/FocusGroup.js

```diff
@@ -29,11 +29,13 @@ type FocusGroupProps = {|
 
 const {useRef} = React;
 
-function focusGroupItem(cell: ReactScopeMethods, event: KeyboardEvent): void {
-  const tabbableNodes = cell.getScopedNodes();
-  if (tabbableNodes !== null && tabbableNodes.length > 0) {
-    tabbableNodes[0].focus();
-    event.preventDefault();
+function focusGroupItem(cell: ReactScopeMethods, event?: KeyboardEvent): void {
```

Comment:
I guess `event` is sometimes nullable but it was missed before?

Reply:
Well spotted!